src/Code4Dependency.php

```
@@ -0,0 +1,166 @@
+        return implode(PHP_EOL, $lines);
+    }
+
+    /**
+     * @param array<string> $lines
+     */
+    private function addSetterCode(array &$lines): void
+    {
+        $newInstance = ($this->prop)($this->dependency, 'newInstance');
+        // class name
+        /** @var class-string $class */
+        $class = ($this->prop)($newInstance, 'class');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$class'. (undefined)

(UnusedLocalVariable)